nemo/backends/pytorch/nm.py

```diff
@@ -129,28 +134,35 @@ def num_weights(self):
         return sum(p.numel() for p in self.parameters() if p.requires_grad)
 
 
-class NonTrainableNM(NeuralModule):
+class NonTrainableNM(NeuralModule, nn.Module):
```

Comment:
NonTrainableNM inheriting from nn.Module? I do not understand this...

Reply:
![TheHierarchy](https://user-images.githubusercontent.com/7504104/79812665-77ced780-832e-11ea-9b88-88e71643d02d.png)

Reply:
This is what I propose. Bonus: get rid of overloading `__call__` :)

Reply:
Is overriding __call__ method the only issue why this is needed?

Reply:
Not only. The second reason is the beauty. :)

Reply:
I like symmetry too, but I do not think this is a good idea here...